### DISCUSSION TITLE
TAX ID checkout field

### DISCUSSION THREAD

**Discussion by PawcioZG** [woocommerce/woocommerce#49215] [Checkout Flow]
Hello, Is there any way to add additional TAX ID field ad the checkout page of woocommerce? I have tried many adds and code but still nothing:(

**Comment by senadir** [comment]
You can follow the tutorial/code examples here to add that

https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/additional-checkout-fields.md

**Comment by PawcioZG** [comment]
Thank you for your support BRPozdrawiamPaweł SzczęsnyWiadomość napisana przez Seghir Nadir ***@***.***> w dniu 22.07.2024, o godz. 17:50:﻿
You can follow the tutorial/code examples here to add that
https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/additional-checkout-fields.md
